test/unit/auth/auth.spec.ts

```diff
@@ -482,6 +482,25 @@ AUTH_CONFIGS.forEach((testConfig) => {
           .should.eventually.be.rejectedWith('Decoding Firebase ID token failed');
       });
 
+      it('should be rejected with checkRevoked set to true and corresponding user disabled', () =>  {
```

Comment:
Can you add 1 more test when both token is revoked and user is disabled. We should throw user disabled error as we check that first.